cmake/.gitignore

```diff
@@ -0,0 +1 @@
+**/build
```

Comment:
Should not be necessary

Reply:
Yeah I just thought it was a nice to have to help keep workspaces clean. Doesn't do much otherwise.